app/views/layouts/application.html.erb

```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
   <title>CarDealershipSm</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
:ok_hand: But *it is not related* with this commit :P 

Be careful with this such of thing :|